src/commands/Misc/randreddit.ts

```diff
@@ -60,22 +62,22 @@ export default class extends SkyraCommand {
 		try {
 			parsed = error.toJSON() as RedditError;
 		} catch {
-			throw message.language.tget('SYSTEM_PARSE_ERROR');
+			throw message.language.get('SYSTEM_PARSE_ERROR');
 		}
 
 		switch (parsed.error) {
 			case 403: {
-				if (parsed.reason === 'private') throw message.language.tget('COMMAND_RANDREDDIT_ERROR_PRIVATE');
-				if (parsed.reason === 'quarantined') throw message.language.tget('COMMAND_RANDREDDIT_ERROR_QUARANTINED');
+				if (parsed.reason === 'private') throw message.language.get('COMMAND_RANDREDDIT_ERROR_PRIVATE');
+				if (parsed.reason === 'quarantined') throw message.language.get('COMMAND_RANDREDDIT_ERROR_QUARANTINED');
 				break;
 			}
 			case 404: {
-				if (!('reason' in parsed)) throw message.language.tget('COMMAND_RANDREDDIT_ERROR_NOT_FOUND');
-				if (parsed.reason === 'banned') throw message.language.tget('COMMAND_RANDREDDIT_ERROR_BANNED');
+				if (!('reason' in parsed)) throw message.language.get('COMMAND_RANDREDDIT_ERROR_NOT_FOUND');
```

Comment:
```suggestion
		if(!Reflect.has(parsed, 'reason')) throw message.language.get('COMMAND_RANDREDDIT_ERROR_NOT_FOUND');
```

Reflect API = cleaner

Reply:
nvm, TS complains on the next line when changing it to that.